src/automation/run_3day_inventory.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -l
+# This script is for running the automated inventory for 
+# the past 3 days to get updates for lagged data and plotting
+# the new data.
+
+SATINFO_DIR=/home/Jessica.Knezha/obs-inventory/build_gsinfo/satinfo
+OUTPUT_LOC=/home/Jessica.Knezha/inventory-figures
+WORK_DIR=/lustre/home/work/inventory-work
+
+cd $(dirname $0)
+
+source ../../obs_inv_utils_pw_cloud.sh
+
+#run inventory 
+python3 auto_inventory.py -cat atmosphere -ago 3 -n_jobs 50 -work_dir $WORK_DIR 
+
+#run all plots in parallel
+python3 ../plotting/plot_mysql_dir_sensor_sat.py --sidb $SATINFO_DIR -o $OUTPUT_LOC &
+python3 ../plotting/plot_mysql_sensor.py -o $OUTPUT_LOC 
+python3 ../plotting/plot_mysql_sensor_sat_amv.py --sidb $SATINFO_DIR -o $OUTPUT_LOC &
+python3 ../plotting/plot_mysql_sensor_sat_geo.py --sidb $SATINFO_DIR -o $OUTPUT_LOC 
+python3 ../plotting/plot_mysql_sensor_sat_gps.py --sidb $SATINFO_DIR -o $OUTPUT_LOC &
+python3 ../plotting/plot_mysql_typ.py -o $OUTPUT_LOC 
```

Comment:
I am curious to the logic of some of the calls running in the background and some are not?

Reply:
I found that it ran more efficiently when there was only two at a time, probably due to IO overload and memory overload from processing all the SQL info. It also made it easier to read the output for errors.